src/Hamiltonians/abstract.jl

```diff
@@ -232,6 +232,69 @@ function build_sparse_matrix_from_LO(
     end
 end
 
+"""
+    basis = build_basis_only_from_LO(
+        ham, add; cutoff, filter=nothing, sort=false, kwargs...
+    )
+
+Get a vector of all basis element of a linear operator `ham` that are reachable (via 
+non-zero matrix elements) from the address `add`. Does not compute or return the matrix.
+
+Providing an energy cutoff will skip addresses with diagonal elements greater
+than `cutoff`. Alternatively, an arbitrary `filter` function can be used instead. These are
+not enabled by default.
+
+Setting `sort` to `true` will sort the basis. Any additional keyword arguments
+are passed on to `Base.sortperm`.
```

Comment:
```suggestion
are passed on to `Base.sort!`.
```
See comment at the bottom.